src/js/components/Drop/StyledDrop.js

```diff
@@ -88,7 +88,7 @@ const StyledDrop = styled(Box)`
     marginStyle(
       props.theme,
       props.alignProp,
-      props.theme.global.drop.margin,
+      props.margin || props.theme.global.drop.margin,
```

Comment:
```suggestion
      props.theme.global.drop.margin,
```
It doesn't seem like we need to pass this through here because it is already passed to the function below on line 93?

Reply:
I was getting an error when I tried removing here. I can look further though if we want to remove
<img width="1173" alt="Screenshot 2025-04-04 at 5 27 05 PM" src="https://github.com/user-attachments/assets/136b0d28-886f-423f-8740-5ffc9447af20" />